src/components/Carousel/Carousel.js

```diff
@@ -47,61 +44,85 @@ class Carousel extends Component {
   onResize() {
```

Comment:
pls remove here event listener, that you've added in componentDidMount

Reply:
If i understand correctly, when I remove this eventListener, the component stops showing the exact amount of cards (doesn't work correctly).

did you mean removing `window.addEventListener('resize', this.onResize)` from componentDidMount?

Reply:
no i mean, that every subscription accomplished in didMount should be removed in componentWillUnmount
just put window.removeEventListener there

Reply:
this is already there